### PR TITLE
added & updated files for dark theme

### LINKED ISSUE #123: Add a dark theme for night
We should add a toggle for users to switch between dark and light mode.

### PR DESCRIPTION
**Pull request by Shivansh-yadav13**
Fixes #123 

## Proposed Changes

  - added header to display the button 
  - updated index.css
  - updated app.js

## Screenshots
ss1
ss2

There is one problem, I am not sure how to change the color of the links of GitHub, and also it changes to white when visiting the profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/281"></a>

### CODE REVIEW

**Review by Jassi10000 — COMMENTED** [review]
Looks good @Shivansh-yadav13 🤘

**Comment by Panquesito7** [comment]
It would be nice if you could fix the GitHub logo looking very dark. 🙂

File: src/Components/Header.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+function Header() {
+  const ThemeChange = () => {
+    const theme = document.getElementsByTagName('BODY')[0].style
+    if (theme.backgroundColor === 'black') {
+      theme.backgroundColor = 'white'
+      theme.color = 'black'
+    } else {
+      theme.backgroundColor = 'black'
+      theme.color = 'white'
```
Comment (eddiejaoude) [inline src/Components/Header.js:11]: Probably better to use `dark` and `light`

File: src/index.css [hunk 2]
```diff
@@ -31,3 +31,13 @@ code {
...
+
+.btn-theme {
+  width: 30px;
+  height: 30px;
+
+  border: none;
+  border-radius: 50%;
+
+  background-color: grey;
+}
```
Comment (eddiejaoude) [inline src/index.css:43]: Probably better to go in a component style like `Header.css`, but even better could we use PrimeReact components

- Buttons https://www.primefaces.org/primereact/showcase/#/button
- Toogles https://www.primefaces.org/primereact/showcase/#/togglebutton

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks good, but use more PrimeReact features than custom code

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Consider resolving the merge conflicts [**here**](https://github.com/EddieHubCommunity/LinkFree/pull/281/conflicts). It will be useful before merging.

File: src/index.css (hunk 2 above)
Comment (Shivansh-yadav13) [reply src/index.css:43]: If we are using PrimeReact then I think using switches will look better?

- https://www.primefaces.org/primereact/showcase/#/inputswitch

**Comment by Shivansh-yadav13** [comment]
Can I make a new clean pull request? I have fixed everything

File: src/index.css (hunk 2 above)
Comment (eddiejaoude) [reply src/index.css:43]: Great 👍

**Pull request closed without merging**